portfolio/src/main/webapp/comments.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+    
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    
+  </head>
+  <body>
+     
```

Comment:
nit: could you please remove extraneous whitespace in this file?

Reply:
I see below that there's some logic to how you've decided to use whitespace. Here's a way to think about it:

1) whitespace should be used to separate semantically distinct blocks. Line 3 currently does that, but line 9 does not (the closing tag has the most relation to the block above.
2) whitespace should only be used to separate when it gives you concrete value. If you've got 2-5 distinct blocks separated by whitespace, that makes it clearer. If you've got a high ratio of whitespace to non-whitespace, that doesn't.
3) The most frequent cases of whitespace making sense are _between_ tags, very rarely would you want whitespace at the beginning or end of a tag